oauth2client/contrib/locked_file.py

```diff
@@ -347,10 +189,15 @@ def __init__(self, filename, mode, fallback_mode, use_native_locking=True):
         """
         opener = None
         if not opener and use_native_locking:
-            if _Win32Opener:
+            try:
+                from oauth2client.contrib._win32_opener import _Win32Opener
```

Comment:
Basically the opener classes depend on two exceptions and one helper method defined in `locked_file.py`.